mapfishapp/src/main/webapp/app/js/GEOR_wmcbrowser.js

```diff
@@ -438,7 +455,9 @@ GEOR.wmcbrowser = (function() {
             // create array of unique keywords
             keywords = [];
             var K = {};
-            Ext.each(GEOR.config.CONTEXTS, function(c) {
+            var storeData = filterContexts(GEOR.config.CONTEXTS.slice(0));
```

Comment:
Don't understand what you mean here, filterContexts modify the array here also, in order to filter keywords, maybe I should use a class attribute ?